src/pdmat.jl

```diff
@@ -65,8 +65,8 @@ end
 
 ### quadratic forms
 
-quad(a::PDMat, x::StridedVector) = dot(x, a * x)
-invquad(a::PDMat, x::StridedVector) = dot(x, a \ x)
+quad(a::PDMat, x::AbstractVector) = dot(x, a * x)
```

Comment:
Ah, you're right. It looks like the definition of `*` is
```julia
*(a::PDMat, x::StridedVecOrMat) = a.mat * x
```
where the `mat` field is `<:AbstractMatrix`. So I suppose one could end up with a multiplication that doesn't make sense even with `x::StridedVecOrMat` depending on what `a.mat` is. To that end, perhaps the signature for `*` should be widened as well, and we can just let the type mismatch case fall back to a `MethodError` in `*`?